sdk/include/vcruntime/eh.h

```diff
@@ -24,10 +28,10 @@ typedef void (__cdecl *unexpected_handler)(void);
 struct _EXCEPTION_POINTERS;
 typedef void (__cdecl *_se_translator_function)(unsigned int,struct _EXCEPTION_POINTERS *);
 
-_CRTIMP __declspec(noreturn) void __cdecl terminate(void);
+_CRTIMP __declspec(noreturn) void __cdecl terminate(void) throw();
```

Comment:
std::terminate is noexcept in C++11, is there any chance these should be as well (I realize throw() is basically the same thing).